Doc/whatsnew/3.7.rst

```diff
@@ -604,3 +604,24 @@ CPython bytecode changes
 
 * Added two new opcodes: :opcode:`LOAD_METHOD` and :opcode:`CALL_METHOD`.
   (Contributed by Yury Selivanov and INADA Naoki in :issue:`26110`.)
+
+
+Documentation
+=============
+
+.. _whatsnew37-pep545:
+
+PEP 545: Python Documentation Translations
+------------------------------------------
+
+:pep:`545` describes the process to translate Python documentations,
```

Comment:
Perhaps change this:

`describes the process to translate Python documentations, and two translation has been added:`

to:

`describes the process to translate Python documentation, and two translations have been added:`

Summary:

- s/documentations/documentation/
- s/translation/translations/
- s/has/have/

Reply:
Thanks!